pkg/netdevice/vdpa.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netdevice
+
+import (
+	"github.com/golang/glog"
+
+	vdpa "github.com/k8snetworkplumbingwg/govdpa/pkg/kvdpa"
```

Comment:
This entire `type vdpaDevice` is behind `types.VdpaDevice` interface that gets mocked for UT. Are you suggesting adding another interface to do a layer of UT only on `vdpa.go`?

Reply:
we can add an interface that eposes kvdpa public methods so they can later be mocked and UT if needed.

like:
 https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/7488066fa720cb32ae77dd412fa84088f3ebac10/pkg/utils/sriovnet_provider.go

https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/7488066fa720cb32ae77dd412fa84088f3ebac10/pkg/utils/netlink_provider.go

adding UT for `vdpa.go` would certainly be nice :) as it would cover the methods in this file.